addons/survey/views/survey_templates.xml

```diff
@@ -400,7 +400,7 @@
                    data-widget="datetime-picker" data-widget-type="date"
                    t-att-data-min-date="question.validation_min_date"
                    t-att-data-max-date="question.validation_max_date"/>
-            <div t-if="not survey_form_readonly" class="position-absolute input-group-text text-primary border-0 bg-transparent p-0 end-0 pe-none"><i class="fa fa-calendar"></i></div>
+            <div t-if="not survey_form_readonly" class="position-absolute input-group-text text-primary border-0 bg-transparent p-0 end-0"><i class="fa fa-calendar"></i></div>
```

Comment:
Instead of *removing* `pe-none`, _add_ `p-1`: this will be stable, fix the issue **and fix the design**.

(btw that design is something that the design team normalized in master, not sure if merged or not yet already, we'll need to review it)

Reply:
> fix the issue

Which issue?

> fix the design

Do you mean the fact that the calendar is floating a bit right of the border-less input? Isn't that on purpose :thinking: ?
I actually did not know about the survey layout and thought it was nice (on top of being unrelated with the current fix)

I'd not add p-1 then

About pe-none, yeah, we might as well take 0 risk and leave it since we remove it in JS anyway, even if people should not xpath such classes, I'll keep those

Reply:
> Which issue?

The one I thought you were fixing ^^ When clicking part of the icon, it does not nothing (because with the p-1, what's below part of the icon is not the input)**

> Do you mean the fact that the calendar is floating a bit right of the border-less input? Isn't that on purpose 🤔 ?

If it's on purpose, I suggest the dev (and you) to fetch medical attention :joy: On top of the mentioned bug, it's obviously totally random: the spacing could be of any value and the icon be of any width :grimacing: Nah but it's of course not on purpose. Especially since in a different theme it means it could be floating even far away from the input just because the theme increased the general spacing :man_facepalming: I tested to be sure, it's actually `top-0` that is missing too, so it needs `top-0` and `p-1`.

Without:
![image](https://github.com/odoo/odoo/assets/10338094/e4e004e9-cd0e-4709-9085-070a6131f706)

With:
![image](https://github.com/odoo/odoo/assets/10338094/e40a6e6c-458d-4fca-a625-2d36295f48eb)

Without in a custom theme which increased the general spacing by a small amount:
![image](https://github.com/odoo/odoo/assets/10338094/f3e5a736-d195-41b3-b844-86e58f7287d4)

**: if you are somehow not convinced by the design stuff ( :exploding_head: ), then be convinced by the fact that in the "custom theme" example, half of the icon is doing nothing :pray:

Reply:
Let's forget about this part: after this PR, it will only be about the design issue => it should thus be an independent fix anyway. And I'll probably ignore survey issues.